web/index.js

```diff
@@ -4,7 +4,9 @@ var options = {
   host: process.env.HOST,
   port: process.env.PORT,
   loginAPI: process.env.LOGINAPI,
-  oauth_clients: process.env.OAUTH_DB ? JSON.parse(process.env.OAUTH_DB) : []
+  oauth_clients: process.env.OAUTH_DB ? JSON.parse(process.env.OAUTH_DB) : [],
+  cookieSecret: process.env.COOKIE_SECRET,
+  secureCookies: process.env.SECURE_COOKIES === true
```

Comment:
should be `=== "true"` since everything that comes through on `process.env` is a string